code/game/mecha/equipment/tools/mining_tools.dm

```diff
@@ -84,6 +84,8 @@
 	if(ishuman(target))
 		var/mob/living/carbon/human/H = target
 		H.apply_damage(drill_damage, BRUTE, "chest")
+		if(target.stat == DEAD)
+			target.gib()
```

Comment:
this probably takes args now but i dont know what those are because searching for procs in git is a bitch